templates/project/.github/workflows/qa.yaml.twig

```diff
@@ -23,6 +23,14 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
+{# Remove when removing 3.x for the managed branches of block-bundle #}
+{% if project.repository.name == 'SonataBlockBundle' and branch.name == '3.x' %}
```

Comment:
Maybe it's better to remove the check, to add this environment variable in every build
since I need it too: https://github.com/sonata-project/SonataDoctrineORMAdminBundle/pull/1170

Reply:
IMO on the doctrineOrm we should first see why it is needed